006_plug_and_play_prior/toto.ipynb

```diff
@@ -0,0 +1,705 @@
7sRJnYiI/BYndSdeficiIgoQPnumHhwcLAav6rOFOqTgkkQTItKGa6S/BKWAki24JoXTpLFI628LnEnbOTIy0trepk0ba3tMTIy1PS4uzqMtNja2yX0BuURwaWmptb2srMyjbf/+/da+hw8ftrZLwczq6mqPNm0wSNr30v60tUtlfCWacsDaEJ50vGmCqZpyqA311/SVApjaUKEmbOgtmoCatgStrV1aR02oriXwTN3JZyd1IiKixnBSd+LldyIiogDBM3UiIvJbPFN34qRORER+i5O6Eyd1IiLyW5zUnXx2Ug8KCmpy+t1GSgZL7ZqSoNK4tOOVEsa29HtoaKi1r9SuSTpLyXopod6uXTtre8eOHa3tCQkJHm0dOnRQPaeUfj9w4IC1/fvvv/do27Vrl7Xvt99+a20/dOiQtf3o0aNNagPk5LYtQQ/IqWHbsSXte+lYlo5P27EiHT/a1L70hmlbf6nkr3QXgtQuJdo1tG/0mjS/t8Zia5fWXTquvHHXj6090CZKf+KzkzoREVFjeKbuxEmdiIj8Fid1J97SRkREFCB4pk5ERH6LZ+pOnNSJiMhvcVJ38tlJPSIiwiOpLaV9beltaUd5o6a1Nv0upcujo6Ob3F9K7UvLlvrb6rlLNdulJHrnzp2t7Z06dbK2R0VFebR5K7ktpX1t7T169LD2lfaDLUEP2BP3FRUV1r5SKl77PQGa7wPQHp+auy2k40qTjAbs+0faVtr2EydOeLRJdxtoviMB0CXRtfXwtaRjyEYat7QMW1pes+6BNlH6E5+d1ImIiBrDM3UnTupEROTXAm1ibg6m34mIiAIEJ3UiIvJbdZffm/M4G3PmzEFKSgrCw8ORlpaGzz//vMH+xcXFSEtLQ3h4OLp374558+Y5fl5QUACXy+XxsGVEGsJJnYiI/FZrTOorVqzAxIkTMXXqVGzZsgWDBw/GiBEjsG/fPmv/PXv24MYbb8TgwYOxZcsWPPXUU3j00Ufx9ttvO/rFxMSgtLTU8ZACsRKf/Uw9MjLSI4FrS24D9vSulNyW0rsRERHWdttypLS4tPG1CXVbAlxKLmtTyrYkum37nQ1NLW7t+kipeKmGvG09pb622vSArq68lJSXUvFSGltz14a0raT9adsmgP341H6ngHa/2UiJc6nuv7TNDx486NF25MgRa9/Dhw9b248dO6Zqtx373qgffzbtNlLKXdveVOfyM+7WCMrNmjUL999/P8aNGwcAyM/Px+rVqzF37lzk5eV59J83bx66du2K/Px8AECvXr2wceNGvPjii7j99tvd/VwuFxITE89uRf4Pz9SJiOi8V1lZ6XhIf3jX1NRg06ZNyMzMdLRnZmZi7dq11t9Zt26dR//hw4dj48aNOHnypLvt6NGj6NatG7p06YKbbroJW7ZsUa8HJ3UiIvJb3rr8npycjNjYWPfDdsYNAPv370dtba3HFb6EhASUlZVZf6esrMza/9SpU9i/fz8AoGfPnigoKMD777+Pt956C+Hh4Rg0aBC+/vpr1fbw2cvvREREjfHW5feSkhLHR6uNfSxZ/yMKY0yDH1vY+p/ZfvXVV+Pqq692/3zQoEG44oor8Nprr+HVV19twpr8iJM6ERGd92JiYsS81Jk6dOiAoKAgj7Py8vJyMZ+TmJho7R8cHIy4uDjr71xwwQW48sor1WfqqsvveXl5uPLKK9GmTRvEx8fj1ltvxa5duxx9jDHIyclBUlISIiIiMHToUGzfvl01KCIioqY41+n30NBQpKWloaioyNFeVFSEgQMHWn8nIyPDo39hYSHS09PFULMxBlu3bhXLb0tUZ+rFxcUYP348rrzySpw6dQpTp05FZmYmduzY4U7Wzpw5E7NmzUJBQQF69OiBGTNmYNiwYdi1axfatGnT5OeKjY1tcirblqRu166dta/U3rFjR2t7fHy8R5u0HtLOkdZD6m9jq8UMyIlhie3ykLRsTZodkGta25avXYa2Vromcd++fXtru1QT3nasSPXwS0tLre3SvadSityWXJfGJ62nJtEspdylY1m6e0Rajm0/S8lyKf2enJxsbbel4hv6rNNGumtBSvPbxlhVVWXte2Yw6kzeSLlrl9HclLt2HC31XOc6/T558mTcfffdSE9PR0ZGBubPn499+/YhKysLADBlyhR8++23WLp0KQAgKysLr7/+OiZPnowHHngA69atw8KFC/HWW2+5l/nMM8/g6quvRmpqKiorK/Hqq69i69atmD17tmpsqkl91apVjv8vXrwY8fHx2LRpE6699loYY5Cfn4+pU6di1KhRAIAlS5YgISEBy5Ytw0MPPaQaHBERka8ZPXo0KioqMH36dJSWlqJPnz5YuXIlunXrBuDHP+bPvGc9JSUFK1euxKRJkzB79mwkJSXh1VdfddzOdujQITz44IMoKytDbGws+vfvj88++wxXXXWVamzN+ky97h7PujOdPXv2oKyszBHdDwsLw5AhQ7B27VrrpF5dXe24daCysrI5QyIiovNIa32hS3Z2NrKzs60/Kygo8GgbMmQINm/eLC7v5Zdfxssvv3xWYznTWd/SZozB5MmTcc0116BPnz4A/n0pSxP1z8vLc9xGIF1SIyIiqq+1ysT6qrOe1B955BH87W9/c3wmUEcT9Z8yZQoOHz7sfpSUlJztkIiIiM5rZ3X5/Ve/+hXef/99fPbZZ+jSpYu7va68XVlZmSOx11DUPywszBq+CQoK8gjZSCE3W2DIFnADgK5du1rb6z4Lqc92i4MURJL+4pPCX1JYzBakkcJsmrAdYB+jND5t6EYTlJPWRyL9USg9p4123NJz2sqqSsebdLuKVK1KE6LShAQbojkmpPFJ20oKytleQ1LwT2qXjiFbOeAz36fOJF1B/Ne//mVtLy8vt7bbQn51RUXqO3ToUJOXAeiDdZq+mgCd1Ne2HwI9KOfLVGfqxhg88sgjeOedd/DJJ58gJSXF8fOUlBQkJiY6ovs1NTUoLi4Wo/5ERERn
```

Comment:
It says "posteriors" but I only see samples from one posterior. Did you mean to show samples for both?  Also, please explain what the reader is to look at/learn from seeing these samples.

Reply:
As we decided to remove different denoisers, so we have only one posterior now